texts/formatted_medtner.txt

```diff
@@ -0,0 +1,1665 @@
+	"*Taneyev"
+"<among>"
+	"among" pr
+"<others>"
+	"others" prn ind mf pl
+"<.>"
+	"." sent
+"<Despite>"
+;	"despite" adv
+	"despite" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<conservative>"
+	"conservative" adj
+;	"conservative" n sg
+"<musical>"
+	"musical" adj
+;	"musical" n sg
+"<tastes>"
+	"taste" n pl
+;	"taste" vblex pres p3 sg
+"<,>"
+	"," cm
+"<Medtner>"
+	"*Medtner"
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<compositions>"
+	"composition" n pl
+"<and>"
+	"and" cnjcoo
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<pianism>"
+	"*pianism"
+"<were>"
+;	"be" vbser past p1 pl
+;	"be" vbser past p2 sg
+;	"be" vbser past p2 pl
+	"be" vbser past p3 pl
+"<highly>"
+	"highly" adv
+"<regarded>"
+	"regard" vblex pp
+;	"regard" vblex past
+"<by>"
+;	"by" adv
+	"by" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<contemporaries>"
+	"contemporary" n pl
+"<.>"
+	"." sent
+"<To>"
+	"to" pr
+"<the>"
+	"the" det def sp
+"<consternation>"
+	"consternation" n sg
+"<of>"
+	"of" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<family>"
+;	"family" adj
+	"family" n sg
+"<,>"
+	"," cm
+"<but>"
+	"but" cnjcoo
+;	"but" pr
+"<with>"
+	"with" pr
+"<the>"
+	"the" det def sp
+"<support>"
+	"support" n sg
+;	"support" vblex inf
+;	"support" vblex pres
+;	"support" vblex imp
+"<of>"
+	"of" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<former>"
+	"former" adj
+;	"former" adv
+"<teacher>"
+	"teacher" n sg
+"<Taneyev>"
+	"*Taneyev"
+"<,>"
+	"," cm
+"<he>"
+	"prpers" prn subj p3 m sg
+"<soon>"
+	"soon" adv
+;	"soon" adj sint
+"<rejected>"
+;	"reject" vblex pp
+	"reject" vblex past
+"<a>"
+	"a" det ind sg
+"<career>"
+;	"career" adj
+	"career" n sg
+"<as a>"
+	"as a" pr
+"<performer>"
+	"performer" n sg
+"<and>"
+	"and" cnjcoo
+"<turned>"
+;	"turn" vblex pp
+	"turn" vblex past
+"<to>"
+	"to" pr
+"<composition>"
+	"composition" n sg
+"<,>"
+	"," cm
+"<partly>"
+	"partly" adv
+"<inspired>"
+;	"inspire" vblex past
+	"inspire" vblex pp
+"<by>"
+;	"by" adv
+	"by" pr
+"<the>"
+	"the" det def sp
+"<intellectual>"
+	"intellectual" adj
+;	"intellectual" n sg
+"<challenge>"
+	"challenge" n sg
+;	"challenge" vblex inf
+;	"challenge" vblex pres
+;	"challenge" vblex imp
+"<of>"
+	"of" pr
+"<Ludwig>"
+	"Ludwig" np ant m sg
+"<van>"
+	"van" n sg
+"<Beethoven>"
+	"Beethoven" np cog sg
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<late>"
+;	"late" adv
+	"late" adj sint
+"<piano>"
+;	"piano" adv
+	"piano" n sg
+"<sonatas>"
+	"sonata" n pl
+"<and>"
+	"and" cnjcoo
+"<string>"
+	"string" n sg
+;	"string" vblex inf
+;	"string" vblex pres
+;	"string" vblex imp
+"<quartets>"
+	"quartet" n pl
+"<.>"
+	"." sent
```

Comment:
I've evaluated up to here.  Make sure there aren't any other things missing analyses before resubmitting.